src/test/java/com/epam/tc/steps/AssertionSteps.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.tc.steps;
+
+import com.epam.tc.dto.BoardDto;
+import com.epam.tc.dto.LabelDto;
+import com.epam.tc.dto.ListDto;
+import com.epam.tc.tests.TrelloFunctionTests;
+import org.assertj.core.api.SoftAssertions;
+
+public class AssertionSteps {
+    private SoftAssertions softAssertions = new SoftAssertions();
+
+    public void verifyThatBoardExist(String expectedBoardName) {
+        BoardDto [] boardDtos = ActionSteps.getBoards();
+        boolean check = false;
+        for (BoardDto ld : boardDtos) {
+            if (ld.getName().equals(expectedBoardName)) {
+                check = true;
+                break;
+            }
+        }
+        softAssertions.assertThat(check).isTrue();
```

Comment:
You can try to use` asserThat(...).contains(...)` to make this assertion simpler